app/components/opplysning-forslag/OpplysningForslag.tsx

```diff
@@ -38,10 +34,10 @@ export function OpplysningForslag(props: IProps) {
         behandlingen. Se nøkkelpunkter for å få en oversikt.
       </BodyLong>
 
-      <Table className={"table--subtle-zebra"}>
-        <Table.Body>
+      <Table zebraStripes>
```

Comment:
Hvorfor er denne byttet ut?
Vi bruker "table--subtle-zebra" i alle andre tabeller i løsningen. Er det noen grunn til at denne skal være annerledes?

Reply:
Brukte <HeaderCel scope="row"> og da ble subtle-zebra litt ødelagt (Stripsene fulgte ikke hele linjen). Aksel-docs brukte bare zebraStripes og det funket bedre.

Reply:
Vi overstyrer en del av aksel sin CSS for å tilpasse den interne flater. Siden du har brukt Table.HeaderCell må du utvide CSSen for subtle-zebra i global.css til å håndtere th i tillegg til td som er der fra før.